webui/react/src/e2e/fixtures/user.fixture.ts

```diff
@@ -24,13 +25,16 @@ export class UserFixture {
     this.userManagementPage = new UserManagement(page);
   }
 
-  async fillUserForm({ username, displayName, isAdmin }: UserArgs): Promise<void> {
+  async fillUserForm({ username, displayName, isAdmin, password }: UserArgs): Promise<void> {
     if (username !== undefined) {
       await this.userManagementPage.createUserModal.username.pwLocator.fill(username);
     }
     if (displayName !== undefined) {
       await this.userManagementPage.createUserModal.displayName.pwLocator.fill(displayName);
     }
+    if (password !== undefined) {
```

Comment:
I believe tests are failing because the UI doesn't allow blank passwords on edit. Is that OK? Do users need to update their password whenever they edit? That seems like it should only be required on creation to me.


If we are sure about this behavior then the edit test needs an update.
Line ref: https://github.com/determined-ai/determined/blob/985daeb7056cc9343ab8614d5fb4dc31b549855e/webui/react/src/e2e/tests/userManagement.spec.ts#L114

Reply:
No, I agree, the intent was that a blank password on the edit modal should mean "keep the password the same" and it shouldn't be required if the user exists already.